docs/usage.rst

```diff
@@ -226,6 +226,18 @@ each scenario instead of flushing the entire database. Tests run much quicker,
 however HTTP server is not started and therefore web browser automation is
 not available.
 
+``--test-runner``
+************
+
+Allows to use custom test runner, provided as dotted path as a value of ``--test-runner``
+option, for instance:
+
+.. code-block:: bash
+
+    $ python manage.py behave --test-runner myapp.runners.CustomTestRunner
```

Comment:
I feel, you should also describe how such a custom test runner can be integrated via source code (e.g. in `environment.py`, maybe?). We should strive to keep the CLI usage simple, `python manage.py behave` should be able to do it, all the rest should be under version control.

Reply:
Also needed would be a description of how custom test runners are implemented. It may make sense to put all that into a separate, new chapter in the documentation.

Reply:
Custom test runner could leave anywhere in the project, it is just a Python class that has correct interface, like this:
https://github.com/behave/behave-django/blob/master/behave_django/runner.py#L33

Reply:
> Custom test runner could leave anywhere in the project, it is just a Python class that has correct interface, like this:
> https://github.com/behave/behave-django/blob/master/behave_django/runner.py#L33

I'm not sure I understand. Yes, the current "predefined test runners" are somewhat "built-in custom test runners" adhering the "correct interface". -- What do you intend with "could leave anywhere"?

Reply:
Test runner is a thing from `unittest`, not from behave, so why do describe it in behave_django docs?

Reply:
Is this a serious question? Because we're adding a feature to behave-django.

If we add a feature to behave-django, in this case "Implement a custom test runner for use with behave-django", we need to document it. Simple as that.